.travis.yml

```diff
@@ -153,115 +141,3 @@ matrix:
           - $HOME/.gradle/caches/
           - $HOME/.gradle/wrapper/
           - $HOME/.m2
-
-    # Disabled due to https://github.com/square/workflow/issues/223
-    # - language: swift
-    #   name: "Swift - Workflow - Cocoapods (Xcode 10.1)"
-    #   os: osx
-    #   osx_image: xcode10.1
-    #   xcode_workspace: SampleApp.xcworkspace
-    #   xcode_scheme: Workflow-Unit-Tests
-    #   xcode_destination: platform=iOS Simulator,OS=10.1,name=iPad Pro (9.7-inch)
-    #   before_install:
-    #     - yes | gem update --system
-    #     - gem install bundler
-    #     - bundle install
-    #     - cd swift/Samples/SampleApp
-    #     - bundle exec pod repo update
-    #     - bundle exec pod install
-    # Disabled due to https://github.com/square/workflow/issues/201
-    # - language: swift
-    #   name: "Swift - WorkflowUI - Cocoapods (Xcode 10.1)"
-    #   os: osx
-    #   osx_image: xcode10.1
-    #   xcode_workspace: SampleApp.xcworkspace
-    #   xcode_scheme: WorkflowUI-Unit-Tests
-    #   xcode_destination: platform=iOS Simulator,OS=10.1,name=iPad Pro (9.7-inch)
-    #   before_install:
-    #     - yes | gem update --system
-    #     - gem install bundler
-    #     - bundle install
-    #     - cd swift/Samples/SampleApp
-    #     - bundle exec pod repo update
-    #     - bundle exec pod install
-
-    - language: swift
-      name: "Swift - Workflow - CocoaPods (Xcode 10.2)"
-      os: osx
-      osx_image: xcode10.2
-      xcode_workspace: gen/Development/Development.xcworkspace
-      xcode_scheme: Development-Unit-WorkflowTests
-      xcode_destination: platform=iOS Simulator,OS=12.2,name=iPad Pro (9.7-inch)
-      before_install:
-        - yes | gem update --system
-        - gem install bundler
-        - bundle install
-        - bundle exec pod gen Development.podspec
-
-    - language: swift
-      name: "Swift - WorkflowUI - CocoaPods (Xcode 10.2)"
-      os: osx
-      osx_image: xcode10.2
-      xcode_workspace: gen/Development/Development.xcworkspace
-      xcode_scheme: Development-Unit-WorkflowUITests
-      xcode_destination: platform=iOS Simulator,OS=12.2,name=iPad Pro (9.7-inch)
-      before_install:
-        - yes | gem update --system
-        - gem install bundler
-        - bundle install
-        - bundle exec pod gen Development.podspec
-
-    - language: swift
-      name: "Swift - SPM - iOS (Xcode 11.1)"
-      os: osx
-      osx_image: xcode11.1
-      script:
-        - xcodebuild -scheme "Workflow-Package" test -destination "name=iPhone 11"
```

Comment:
@AquaGeek, @amorde: This is the SPM test that I copied over to the Github Actions. Not entirely sure what's happening here, but my assumption was `xcodebuild` is using `Package.swift` to build and test the `Workflow` package.

Reply:
that makes sense 👍